src/pretalx/api/views/event.py

```diff
@@ -24,3 +41,125 @@ def get_object(self):
         if self.request.user.has_perm(self.permission_required, self.request.event):
             return self.request.event
         raise Http404()
+
+
+@api_view(http_method_names=["POST"])
```

Comment:
As reminded in group chat, use constants from `http` module, don't use hardcode for HTTP methods, statuses.

Reply:
thanks, I have updated the code please help me to check it.